doc/specs/stdlib_experimental_error.md

```diff
@@ -1,8 +1,10 @@
-# Catching and handling errors
+---
+title: experimental_error
+---
 
-* [`check` - Checks the value of a logical condition](#check---checks-the-value-of-a-logical-condition)
-* [`error_stop` - aborts the program](#error_stop---aborts-the-program)
+# Catching and handling errors
 
+[TOC]
```

Comment:
The anchor/backlink syntax seems to be different in python markdown, and we have the automatic TOC extension enabled, so I decided to use that rather than manually maintain a TOC.

Reply:
LGTM!